### PR TITLE
Adding The Carpentries reference

### PR DESCRIPTION
**Pull request by malvikasharan**
Updating CoC document to acknowledge The Carpentries reference

### CODE REVIEW

**Review by yochannah — APPROVED** [review]

File: code-of-conduct/README.md [hunk 1]
```diff
@@ -18,8 +18,7 @@
...
+In 2018 and 2019, The Carpentries Code of Conduct was revised to add a summary, straightforward examples of both beneficial and unwanted behaviors, evaluating intent, reporting guidelines, an appeal process, the procedure for following up with a reportee, terminology, Code of Conduct incident response procedure, termed suspension checklist, expanded clauses for conflicts of interest, and committee membership agreement -- all of which have been adopted by the OBF Code of Conduct. Contributors of these revised documents are Ethan White, Kari L. Jordan, Karin Lagesen, Malvika Sharan, Samantha Ahern, and Simon Waldman.
```
Comment (nlharris) [inline code-of-conduct/README.md:22]: Contributors of -> Contributors to
(but this is a trivial change)

**Review by nlharris — APPROVED** [review]

**Review by kipkurui — APPROVED** [review]

File: code-of-conduct/README.md (hunk 1 above)
Comment (hlapp) [inline code-of-conduct/README.md:22]: If you made the linebreak to introduce a paragraph break, you'd have to add an empty line.

**Review by hlapp — APPROVED** [review]

**Comment by nlharris** [comment]
Can this be merged or are there still things that need to be changed?

**Comment by peterjc** [comment]
Some trivial points like a line break aside, good to merge.

**Pull request merged**